data/product.d/fedora-workstation.conf

```diff
@@ -9,6 +9,7 @@ product_name = Fedora
 
 [Payload]
 default_environment = workstation-product-environment
+import_keys_to_rpm = /etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-$releasever-$basearch
```

Comment:
This should be in `fedora.conf`, not `fedora-workstation.conf`.

Reply:
Done.